components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/introspection/IntrospectionResponseBuilderTest.java

```diff
@@ -88,6 +88,7 @@ public void testResposeBuilderWithVal(boolean isActive, int notBefore, int expir
         introspectionResponseBuilder1.setClientId("rgfKVdnMQnJSSr_pKFTxj3apiwYa");
         introspectionResponseBuilder1.setErrorCode("Invalid input");
         introspectionResponseBuilder1.setErrorDescription("error_discription");
+        introspectionResponseBuilder1.setAuthorizedUserType("APPLICATION_USER");
```

Comment:
Shall we use already available constants for this?

Reply:
+1 fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1459/commits/66a23338204aacc1be2b5a3969fa2bd518322e55